packages/client/test/unit/ResendSubscription.test.ts

```diff
@@ -0,0 +1,46 @@
+import 'reflect-metadata'
+import { StreamPartIDUtils } from 'streamr-client-protocol'
+import { fastWallet } from 'streamr-test-utils'
+import { DestroySignal } from '../../src/DestroySignal'
+import { ResendSubscription } from "../../src/subscribe/ResendSubscription"
+import { fromArray } from '../../src/utils/GeneratorUtils'
+import { collect } from '../../src/utils/iterators'
+import { STREAM_CLIENT_DEFAULTS } from './../../src/Config'
+import { StreamrClientError } from './../../src/StreamrClientError'
+import { createMockMessage } from './../test-utils/utils'
+
+const STREAM_PART_ID = StreamPartIDUtils.parse('streamId#0')
+
+describe('ResendSubscription', () => {
```

Comment:
Check logging in Resends2.test.ts with integration test as discussed.

Reply:
Fixed